integration/kubernetes/init.sh

```diff
@@ -155,7 +155,10 @@ echo "Kubelet options:"
 sudo cat /var/lib/kubelet/kubeadm-flags.env
 sudo systemctl daemon-reload && sudo systemctl restart kubelet
 
-kubectl get nodes
+for i in $(seq 4)
+do
+        kubectl get nodes && break || sleep $i
```

Comment:
@jodh-intel while working with Dockerfile delinting I came across an interesting pattern catch by ShellCheck, which is the `A && B || C`. If `A` is true and `B` is false then `C` is executed (in other words, it not always can be used as a replacement of `if A then B else C`). Here is the rule https://github-wiki-see.page/m/koalaman/shellcheck/wiki/SC2015

I see that pattern on my places. Maybe worth mentioning on the shell script advise document?